src/pages/dev/send-tokens/interchain-tokens/create-token.mdx

```diff
@@ -0,0 +1,93 @@
+
+The simplest type of Interchain Token is to create a brand new [Interchain Token](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/InterchainTokenService.sol) available on multiple chains. For a detailed guide on how to create an Interchain Token using the Interchain Token Portal, refer to the four-step tutorial provided [here](https://axelar.network/blog/how-to-create-an-interchain-token-with-axelar-in-4-steps).
+
+- Visit the [Interchain Portal](https://testnet.interchain.axelar.dev)
+- Connect your wallet to the portal
+- Select a source network where you have funds available
+- Choose the option to deploy a new Interchain token
+- Add the required details for your new token
+    - Name
+    - Symbol
+    - Decimals
+    - Amount to mint
+- You can also click the advanced option where you can add an account as `Token Minter` and `Salt` value; otherwise, it gets prefilled with the deployer(connected account) address and a random, uniquely generated salt value
+- Select additional chains for your token's availability and optionally add the token amount to mint on each selected chain.
+
+**Congratulations!** Your Interchain Token is now accessible on multiple blockchain networks. This allows for seamless interaction with your token on any of these networks, utilizing familiar methods like send, transfer, and approve, just as with any standard ERC-20 token.
+
+Furthermore, your token is categorized as a [Interchain Token](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IInterchainTokenStandard.sol). This classification includes the `interchainTransfer` method, allowing you to transfer it between blockchains.
+
+Here is an example of the `interchainTransfer` method.
+
+```solidity
+/**
+ * @notice Initiates an interchain transfer of a specified token to a destination chain.
+ * @dev The function retrieves the TokenManager associated with the tokenId.
+ * @param tokenId The unique identifier of the token to be transferred.
+ * @param destinationChain The destination chain to send the tokens to.
+ * @param destinationAddress The address on the destination chain to send the tokens to.
+ * @param amount The amount of tokens to be transferred.
+ * @param metadata Optional metadata for the call for additional effects (such as calling a destination contract).
+ */
+function interchainTransfer(
+    bytes32 tokenId,
+    string calldata destinationChain,
+    bytes calldata destinationAddress,
+    uint256 amount,
+    bytes calldata metadata,
+    uint256 gasValue
+) external payable
+```
+
+## Create a new custom Interchain Token(s)
+
+If you want more features than the Interchain Token provides, you can create a [custom](../../reference/glossary/#custom-token) token. You may want a customized token to customize specific minting policies, ownership structures, rate limits, or other bespoke functionalities.
+
+To create a new Interchain Token:
```

Comment:
The steps below describe how to deploy an InterchainToken, not a custom token. To deploy a custom token the user needs to
- deploy their custom ERC20, which should include mint/burn implementations
- use `deployTokenManager` with the correct params (mint/burn as the type) on the ITS
- make sure that the service can call mint/burn on the token.
- Do the same on any other chains the user wants to have their token exist at, using the same deployer address and salt.